### PR TITLE
WorkItem Details page

### PR DESCRIPTION
**Pull request by zpawn**
Story https://app.shortcut.com/deskpro/story/154168/azure-devops-details-page

### CODE REVIEW

File: src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      items: [{ title: "Unlink Item" }],
+    });
+    registerElement("azureEditButton", { type: "edit_button" });
+  });
+
+  useDeskproAppEvents({
+    onElementEvent(id) {
+      switch (id) {
+        case "azureMenuButton":
+          if (client && ticketId && itemId) {
```
Comment (AshleyDawson) [inline src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx:47]: best pass ticketId and itemId along with the app element event?

**Review by AshleyDawson — APPROVED** [review]

File: src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx (hunk 1 above)
Comment (zpawn) [reply src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx:47]: Understand. This is an old implementation, Currently, I'm doing as you said.

Comment (zpawn) [reply src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx:47]: I'll fix it in this story https://app.shortcut.com/deskpro/story/154161/azure-devops-improvements

Comment (AshleyDawson) [reply src/pages/WorkItemDetailsPage/WorkItemDetailsPage.tsx:47]: ok, thanks Ilia

**Pull request merged**